pkg/config/release.go

```diff
@@ -37,7 +37,7 @@ const (
 	TemplatesPath = "ci-operator/templates"
 	// ClusterProfilesPath is where profiles are stored in the release repo
 	ClusterProfilesPath = "cluster/test-deploy"
-	// StagingNamespace is the staging namespace in api.ci
+	// StagingNamespace is the staging namespace in app.ci
```

Comment:
That ns doesn't exit on app.ci

Reply:
The constant is used in some of our tools.
I will handle this with another PR to keep the damage isolated.